client/main.lua

```diff
@@ -2,6 +2,69 @@ local config = require 'config.client'
 local sharedConfig = require 'config.shared'
 local VEHICLES = exports.qbx_core:GetVehiclesByName()
 
+---@enum ProgressColor
+local ProgressColor = {
+    GREEN = 'green.5',
+    YELLOW = 'yellow.5',
+    RED = 'red.5'
+}
+
+---@param percent number
+---@return string
+local function getProgressColor(percent)
+    if percent >= 75 then
+        return ProgressColor.GREEN
+    elseif percent > 25 then
+        return ProgressColor.YELLOW
+    end
+
+    return ProgressColor.RED
+end
+
+---@enum StateLabels
+local StateLabels = {
+    OUT = 0,
+    GARAGED = 1,
+    IMPOUND = 2,
+}
+
+---@param state number
+---@return string
+local function getStateLabel(state)
+    if StateLabels.OUT == state then
+        return Lang:t('status.out')
+    elseif StateLabels.GARAGED == state then
+        return Lang:t('status.garaged')
+    elseif StateLabels.IMPOUND == state then
+        return Lang:t('status.impound')
+    end
+
+    return 'Unknown'
```

Comment:
This is an example where I think it's fine and appropriate for the implied else since 'Unknown' is a default value and therefore not as related to each other as the other 3.